corehq/apps/analytics/tasks.py

```diff
@@ -656,31 +656,18 @@ def track_periodic_data():
 
 
 def _email_is_valid(email):
-    if not email or _is_suspicious_email(email):
+    if not email:
         return False
 
     try:
         validate_email(email)
-    except EmailNotValidError as e:
-        logger.warning(str(e))
+    except ValidationError as exc:
+        logger.warning(str(exc))
         return False
 
     return True
 
 
-# These domains provide disposable email addresses which attract scammers
-# AWS Guard Duty triggers alerts for these domains. The below list is likely incomplete --
-# if a Guard Duty alert is triggered for a domain not seen below, please add it
-SUSPICIOUS_DOMAINS = [
-    'mailna.me',
-    'mozej.com'
```

Comment:
Previously these domains were invalid, and did not allow them to be used. With the change you made, is it the case that we will now attempt to send email to that domain if it is used? And will that trigger the Guard Duty alert?

If yes, much as I don't like the idea, maybe we should maintain the blacklist and reject emails with those domains? In any case this blacklist seems like a separate concern from removing `email-validator`.

Reply:
Attempting to send emails to these addresses was never what caused Guard Duty alerts, it was `email-validator`'s DNS queries that caused them. AFAIK, there was never a problem with having emails to these domains. My understanding is that AWS's SES has its own "suspicious email account" filtering mechanism, and that that process has its own workflow that doesn't depend on this code.

@mjriley does this sound right?

Reply:
It sounds like you're saying django's `validate_email` doesn't do a DNS query (validating based solely on the format of the string), and so it doesn't have this problem. If so, that's great!

Reply:
> doesn't do a DNS query

That's correct, @dannyroberts

Reply:
Yes, it's the DNS query to a 'suspicious' domain that was causing the issue. AWS compiles a list of domains like the ones above, that are known to allow temporary email addresses, and alerts when it sees queries to those domains, because it knows hackers tend to use them.